site/_includes/layouts/home.njk

```diff
@@ -51,10 +51,10 @@
       }
      </style>
     {{ heroCard(
-        "Let’s go. It’s Google I/O",
-        "Learn about the latest innovations in Chrome and the web and what we’re doing to help you work smarter, ship faster, and stay inspired.",
-        "Register now",
-         "https://io.google/2023/?utm_source=chrome&utm_medium=embedded_marketing&utm_campaign=hpp_reg_card&utm_content=",
+        "See what’s happening at Google I/O",
+        "Take a look at content from Chrome at Google I/O—introducing new web platform features and improving your development workflow.",
```

Comment:
Maybe intentional, but missing spaces around the `-`?

Reply:
as per style guide, we don't put spaces around it.